server/src/main/java/org/apache/cassandra/sidecar/datahub/SchemaReporter.java

```diff
@@ -113,13 +113,23 @@ protected SchemaReporter(@NotNull IdentifiersProvider identifiersProvider,
     /**
      * Public method for converting and reporting the Cassandra schema
      *
-     * @param cluster a {@link Cluster} to extract Cassandra schema from
+     * @param cluster the {@link Cluster} to extract Cassandra schema from
      */
     public void process(@NotNull Cluster cluster)
+    {
+        process(cluster.getMetadata());
+    }
+
+    /**
+     * Public method for converting and reporting the Cassandra schema
+     *
+     * @param metadata the {@link Metadata} to extract Cassandra schema from
+     */
+    public void process(@NotNull Metadata metadata)
     {
         try (Emitter emitter = emitterFactory.emitter())
         {
-            stream(cluster.getMetadata())
+            stream(metadata)
                     .forEach(ThrowableUtils.consumer(emitter::emit));
```

Comment:
```suggestion
            stream(metadata).forEach(ThrowableUtils.consumer(emitter::emit));
```

Reply:
It's way easier to read *and* put breakpoints in this way.

Giving it another thought, three lines make even more sense, updated.

Reply:
If in 2 lines, the extra indentation at the second line should be removed. 

```suggestion 
            stream(metadata)
            .forEach(ThrowableUtils.consumer(emitter::emit));
```